### PR TITLE
#433: Update documentation

### PR DESCRIPTION
**Pull request by jekuaitk**
https://leantime.itkdev.dk/#/tickets/showTicket/433

* Adds disclaimer regarding setup to README
* Updates coding standard management (check/apply)

`CHANGELOG.md` is not updated as there has not been an official release yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Sets up an API with data from [Serviceplatformen Organisation](https://digitalis
 * [Symfony](https://symfony.com)
 * [API Platform](https://api-platform.com/)
 
+## Disclaimer
+
+The currently suggested setup exposes the API via docker networks, see
+[Using the API from another docker compose setup](#using-the-api-from-another-docker-compose-setup).
+__It is very important that API is not exposed to the public.__
+
```
Comment (rimi-itk) [inline README.md:17]: Then why do we do it, i.e. suggest it? What is it we can't do if we don't expose the API to the public (by default)?

Reword to a more general warning that the API has no authentication methods and should not be exposed on the web.

File: README.md [hunk 2]
```diff
@@ -205,7 +211,26 @@ we decided to adhere to in this project.
...
+   ```sh
+   docker compose exec phpfpm composer coding-standards-apply
+   ```
+
+* Markdown files (markdownlint standard rules)
+
+  ```sh
+  docker run --rm --interactive --tty --volume ${PWD}:/app node:18 yarn --cwd /app install
+  docker run --rm --interactive --tty --volume ${PWD}:/app node:18 yarn --cwd /app coding-standards-apply
```
Comment (rimi-itk) [inline README.md:233]: Consider adding a `node` service in `docker-compose.override.yml` as we usually do for stuff like this.

File: README.md [hunk 3]
```diff
@@ -205,7 +211,26 @@ we decided to adhere to in this project.
...
+
+* PHP files (PHP-CS-Fixer with the Symfony ruleset enabled)
+
+   ```sh
+   docker compose exec phpfpm composer coding-standards-apply
+   ```
+
+* Markdown files (markdownlint standard rules)
+
+  ```sh
```
Comment (rimi-itk) [inline README.md:231]: Inconsistent indentation (2 and 3 spaces before backticks).

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jekuaitk) [reply README.md:17]: Reworded to a more general warning.

File: README.md (hunk 2 above)
Comment (jekuaitk) [reply README.md:233]: Added a `node` service.

File: README.md (hunk 3 above)
Comment (jekuaitk) [reply README.md:231]: Fixed.

**Pull request merged**